Sources/PointFree/PageLayout.swift

```diff
@@ -60,6 +60,16 @@ enum NavStyle {
 }
 
 struct SimplePageLayoutData<A> {
+  enum Style {
```

Comment:
Promoted `NavStyle` to `Style` so we can render pages without the footer (_e.g._, the printable invoice).